install-macos.sh

```diff
@@ -68,6 +70,29 @@ else
     exit 1
 fi
 
+##
+# Check Developer ID
+##
+printf "\033[34m\n* Ensuring package Developer ID matches\n\033[0m"
+
+if pkgutil --check-signature $PKG_PATH | grep -q "$DEVELOPER_ID"; then
```

Comment:
Let's do /usr/bin/grep here instead, since aliasing grep to another search tool that may or may not support the same flags is pretty common.

Reply:
Updated 👍